### PR TITLE
Determine when tasks should be created for Pull Request actions

### PR DESCRIPTION
**Pull request by josephperrott**

### CODE REVIEW

File: shared/firestore-models/util.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  if (configDoc.exists) {
+    const config = configDoc.data() as BranchManagerRepoConfig;
+    return !!config.branches.find(branch => branch.label === label);
+  }
+  return false;
+}
+
+export async function hasTargetedLabel(configId: string, pullRequestRef: firebase.firestore.DocumentReference) {
+  const configRef = await getConfigRef(configId);
+  const configDoc = await configRef.get();
```
Comment (devversion) [inline shared/firestore-models/util.ts]: We often seem to call `getConfigRef` and then just await the actual snapshot. Should we just make it a helper function within `repo-config.ts`?

File: shared/firestore-models/util.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import * as firebase from 'firebase-admin';
+
+import {getConfigRef, BranchManagerRepoConfig} from './repo-config'
+import {BranchManagerPullRequest} from './pull-request'
+
+
+export async function isTargetLabel(configId: string, label: string) {
```
Comment (devversion) [inline shared/firestore-models/util.ts]: Superfluous line above, and maybe add a small comment about what this function does? 

From what does it check the label? etc. it's a bit unclear I think.

File: shared/firestore-models/util.ts (hunk 1 above)
Comment (josephperrott) [reply shared/firestore-models/util.ts]: I initially made a helper for doing that, but it ended up that we had to `getConfigDoc` calls `getConfigRef` so if you want to get both values, you end up calling `getConfigRef` twice and having to wait on both promises.

I am not sure how to handle it to be efficient

Comment (devversion) [reply shared/firestore-models/util.ts]: `getConfigDoc` could also accept a reference optionally, so if you want both values, you just pass the reference to the `getConfigDoc` function.

Comment (josephperrott) [reply shared/firestore-models/util.ts]: Updated as I was misreading what was being talked about here.

File: shared/firestore-models/util.ts (hunk 2 above)
Comment (devversion) [reply shared/firestore-models/util.ts]: One last thing: I'd feel better if the function is called `isConfiguredTargetLabel` or something (now that I understand what it does)

Your choice though, and optional 😄

**Review by devversion — APPROVED** [review]

**Pull request merged**